doc/manipulating_visualizing/data_preparation.rst

```diff
@@ -78,15 +89,15 @@ have to call low-level functions (see e.g. :mod:`nilearn.signal`,
 :class:`NiftiMasker`: loading, masking and filtering
 =========================================================
 
-This section describes how to use the :class:`NiftiMasker` class in
-more details than the previous description. :class:`NiftiMasker` is a
+This section details how to use the :class:`NiftiMasker` class.
+:class:`NiftiMasker` is a
 powerful tool to load images and extract voxel signals in the area
 defined by the mask. It is designed to apply some basic preprocessing
-steps by default with commonly used default parameters. But it is
+steps by default with commonly used parameters. But it is
```

Comment:
You are removing part of the meaning of this sentence as I understand it, i.e. that the default parameters are set to commonly used values. Maybe "with commonly used parameters as defaults"?